api/SVGImageElement.json

```diff
@@ -158,10 +158,10 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "63"
```

Comment:
This (`decoding`) is the most recent addition out of all these, and it checks out: https://bugzilla.mozilla.org/show_bug.cgi?id=1416328